packages/site/src/components/NoMatch.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+
+const styles = {
+  container: {
+    textAlign: 'center'
+  },
+  error: {
+    fontSize: '10em',
+    fontWeight: 'bold',
+    margin: '1em 0 20px'
+  }
+};
+
+const Error = createStyledComponent('p', styles.error);
+const Container = createStyledComponent('div', styles.container);
+
+export default function NoMatch() {
```

Comment:
ah, I thought we still wanted to have 404s for some reason. I can remove all this.